multimodal/src/autogluon/multimodal/data/process_ner.py

```diff
@@ -51,12 +52,15 @@ def __init__(
         """
         self.model = model
         self.prefix = model.prefix
+        self.tokenizer = None
+        self.max_len = max_len
         if self.prefix == NER:
             self.tokenizer = model.tokenizer
-            self.max_len = self.tokenizer.model_max_length
-        else:
-            self.tokenizer = None
-            self.max_len = None
+            if self.tokenizer.model_max_length > max_len:
```

Comment:
Do we need to check `max_len` against None?

Reply:
fixed. max_len is set to 512 by default in the model config.